### PR TITLE
It allows to display addonId and realId

### PR DESCRIPTION
**Pull request by k33g**
eg:
```
Addon fsbucket-acme (id:addon_b423d27f-f99f-46fa-b4e9-8d7d1abe27e7|realId:bucket_c28cf254-0134-407e-a8f1-f51505c08e61) successfully created
```

### CODE REVIEW

**Comment by divarvel** [comment]
Do you need both? It clutters the output

**Comment by k33g** [comment]
I need `realId` when using fsbucket
I take the realId, replace `_` by `-` and generate this json file

```
 [{
   "bucket_host": "${id}-fsbucket.services.clever-cloud.com",
   "folder": "/storage"
 }]
```

so perhaps there are other solutions:

- output on several lines:
  ```
  Addon fsbucket-acme successfully created
  - id:addon_b423d27f-f99f-46fa-b4e9-8d7d1abe27e7
  - realId:bucket_c28cf254-0134-407e-a8f1-f51505c08e61
  ```
- a command to get the `realId` from the `id`
- or a command to get all informations of addon(s)
- or something I didn't see

and I can't make a choice between `realId` and `addon.id` because I need `addon.id when deleting

**Comment by divarvel** [comment]
I see. constructing bucket host this way is fragile as the actual domain is an implementation detail.
I can add bucket_host to the environment variables exposed by the fs bucket so you don't have to craft it. That way you won't need `realId`

**Comment by k33g** [comment]
@divarvel yes it's a very good solution for me
I will update the PR to only display addonId

**Comment by k33g** [comment]
@divarvel if you prefer, I can delete the PR and create a new one

**Comment by divarvel** [comment]
you can update the PR

**Comment by divarvel** [comment]
The change has been deployed on the FS Bucket addon, new buckets provide a BUCKET_HOST env variable containing the host.

File: src/commands/addon.js [hunk 1]
```diff
@@ -68,9 +68,9 @@ var create = addon.create = function(api, params) {
 
   s_result.onValue(function(r) {
     if(linkTo) {
-      Logger.println("Addon " + name + " (" + r.realId +") successfully created and linked to the application");
+      Logger.println("Addon " + name + " (id:" + r.id +") successfully created and linked to the application");
```
Comment (divarvel) [inline src/commands/addon.js]: Teeny tiny nitpick: can you add a space after the colon?

Comment (k33g) [inline src/commands/addon.js]: ok

**Pull request merged**